ec2/ec2-create-instance.sh

```diff
@@ -156,9 +161,9 @@ echo "Instance ID: "$INSTANCE_ID
 DESTROY_CMD="aws ec2 terminate-instances --instance-ids $INSTANCE_ID"
 echo "When you are done, please terminate your instance with:"
 echo "$DESTROY_CMD"
-echo "giving instance 60 seconds to wake up..."
+echo "giving instance 90 seconds to wake up..."
 
-sleep 60
+sleep 90
```

Comment:
We always seem to need to bump this. Sad!

Reply:
The best things come to those who wait.